public/js/include/bootstrap.min.js

```diff
@@ -1,7 +0,0 @@
-/*!
```

Comment:
@brynbeaudry  I removed these because these aren't necessary. Bootstrap and jquery is integrated to laravel by default. Didn't catch this in your PR.